sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -1168,19 +1168,24 @@ class EntitiesRecognitionTask(DictMixin):
     """EntitiesRecognitionTask encapsulates the parameters for starting a long-running Entities Recognition operation.
 
     :ivar str model_version: The model version to use for the analysis.
+    :ivar str string_index_type: Specifies the method used to interpret string offsets.
```

Comment:
nit: this seems like too many spaces in a tab, it looks. like 8 and we generally want 4

Reply:
Ok, I removed the extra tabs.  This was probably a copy-pasta error.  :)